src/orange/orangeinp/ConvexRegion.hh

```diff
@@ -216,10 +216,48 @@ class GenTrap final : public ConvexRegionInterface
     using VecReal2 = std::vector<Real2>;
     //!@}
 
+    struct TrapFace
+    {
+        real_type hy_{};  // vertical distance between horizontal edges
+        real_type hx_lo_{};  // top horizontal edge
+        real_type hx_hi_{};  // botton horizontal edge
```

Comment:
Since these are public data, omit the trailing `_`, and also use doxygen markup

```suggestion
        real_type hy_{};  //!< half the vertical distance between horizontal edges
        real_type hx_lo_{};  //!< top horizontal edge half-length
        real_type hx_hi_{};  //!< bottom horizontal edge half-length
```